src/calibre/ebooks/metadata/sources/douban.py

```diff
@@ -44,112 +45,6 @@ def get_details(browser, url, timeout):  # {{{
 # }}}
 
 
-def to_metadata(browser, log, entry_, timeout):  # {{{
```

Comment:
I have moved this function into Douban class, because I need to get access to the `options` variable in Douban class.